test-cases/tests/documentation/i13-verify-partial-ag-network-policy.md

```diff
@@ -0,0 +1,25 @@
+# i13 verify partial ag network policy
+
+Sprint #20
+
+## Description
+
+Improve and correct network policy content per QE review
+
+- Merge Request: https://gitlab.cee.redhat.com/RedHatManagedIntegration-documentation/rhmi-docs/-/merge_requests/133
+- Jira ref link: https://issues.redhat.com/browse/RHMIDOC-80
+- Peer contact: Joan Edwards
+
+* Staging doc link: N/A
+  > **NOTE:** The _Staging doc link_ is not to the Customer Portal. The formatting will not match what is on the Customer Portal; however, the steps should be the same.
+
+## Acceptance Criteria
+
+. Test docs/modules/admin-guide/proc_disabling-communication-between-managed-services-project.adoc
```

Comment:
The reviewers will need the full URL to access the (correct branch) for these files.